<!-- -->
<launch>
	<!-- waypoint saver -->
	<arg name="waypointFilePath" default="/home/yimeng/Documents/waypoints/waypoints_outdoor.txt" />
	<arg name="pubrate" default="0.5" />

	<!-- waypoint saver -->
	<node pkg="waypoint_follower" type="saveWaypoints.py" name="waypoint_saver" output="screen">
	    <param name="waypointFilePath" value="$(arg waypointFilePath)" />
	    <param name="pubrate" value="$(arg pubrate)" />
	</node>

</launch>
